### ISSUE TITLE
[WTF] Don't add unneeded context

### ISSUE THREAD

**Issue by peter-gribanov** [jupeter/clean-code-php#15]
https://github.com/jupeter/clean-code-php#dont-add-unneeded-context

What did you try to show in this code?

```php
$car = [
    'carMake'  => 'Honda',
    'carModel' => 'Accord',
    'carColor' => 'Blue',
];

function paintCar(&$car) {
    $car['carColor'] = 'Red';
}
```

What relation has the name of the variable to the prefixes of the keys of the array?
Why do you change the array by reference?
Why do not you use classes to describe the structure?

Maybe you meant something like this:

```php
class Car
{
    public $carMake;
    public $carModel;
    public $carColor;

    public function paint($color)
    {
        $this->carColor = $color;
    }
}
```

and

```php
class Car
{
    public $make;
    public $model;
    public $color;

    public function paint($color)
    {
        $this->color = $color;
    }
}
```

**Comment by hailwood** [comment]
I agree with @peter-gribanov that the class example is better. Although, I feel the function is completely pointless and obscure the point of this example. 

```php
class Car
{
    public $carMake;
    public $carModel;
    public $carColor;

    //...
}
```

And

```php
class Car
{
    public $make;
    public $model;
    public $color;

    //...
}
```

Would suffice.

**Comment by piotrplenik** [comment]
Thx for contributing. 
I agree with you - could you create PR with fix?

**Comment by TomasVotruba** [comment]
Closed by #34, thank you @peter-gribanov
